tools/clang/lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -2894,10 +2894,10 @@ void CGDebugInfo::EmitDeclare(const VarDecl *VD, llvm::dwarf::Tag Tag,
   // Get location information.
   unsigned Line = 0;
   unsigned Column = 0;
-  if (!Unwritten) {
+  // if (!Unwritten) { // HLSL Change - Don't create zero line numbers
```

Comment:
What location is retrieved here for implicit decls that had no location information to begin with (HLSL builtins)?